OntologyMapper/src/README.md

```diff
@@ -113,13 +113,25 @@ A collection of mappings from one or more property names to another property nam
       "OutputPropertyName": "output-property-name",
       "InputPropertyNames": [ "input-property-name1", "input-property-name2" ],
       "IsOutputPropertyCollection": true
+    },
+    {
+      "OutputDtmiFilter": "\w*Space\w*",
+      "OutputPropertyName": "output-property-name",
+      "InputPropertyNames": [ "input-property-name2" ],
+      "IsOutputPropertyCollection": true
     }
+
   ],
   "FillProperties": [
     {
       "OutputDtmiFilter": "*",
```

Comment:
If you make this `.*` you could just make this a regex field rather than having to handle `*` explicitly

Reply:
We'll have to do a data migration for this to happen. Need to roll out this change, then update the current mapping files, roll them out, then remove the * logic to make sure we don't break anything.